pkgs/by-name/cu/cursor-cli/package.nix

```
@@ -0,0 +1,62 @@
+{
+  lib,
+  fetchurl,
+  stdenv,
+  autoPatchelfHook,
+}:
+
+let
+  inherit (stdenv) hostPlatform;
+  version = "2025.08.08-f57cb59";
```

```suggestion
  version = "0-unstable-2025-08-08";
```
Please review the [package versioning guidelines](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#versioning):
- The version attribute _must_ start with a digit.
- If we're tracking a commit from a repository without a version assigned, then the version attribute should be the latest upstream version preceding that commit, followed by `-unstable-` and the date of the fetched commit. The date must be in [ISO 8601](https://en.wikipedia.org/wiki/ISO_8601) "YYYY-MM-DD" format.
  - If a project has no suitable preceding releases, then the latest upstream version in the above schema should be `0`.